styles/Page.module.scss

```diff
@@ -4,41 +4,54 @@
   width: 100vw;
   max-width: $max-width;
   display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
 }
 
-#frame {
-  margin-top: calc(#{$navbar-height} + 100px);
-  margin-left: $padding-horiz;
-  width: $frame-width;
-  height: calc(#{$frame-width} * 2 / 3);
+#background {
   display: flex;
   justify-content: center;
-  align-items: center;
+  min-height: 100vh;
+  background: linear-gradient(rgba(72, 150, 202, 0.2), rgba(237, 209, 71, 0.1));
+  background-attachment: fixed;
 }
 
-#container {
-  margin-top: calc(#{$navbar-height} + 100px);
-  margin-left: $padding-horiz;
-  margin-right: $padding-horiz;
-  display: flex;
-  flex-direction: column;
+#logo-main {
+  width: 200px;
+  height: 200px;
+  margin-top: 50px;
+  background-image: url('../public/logo_transparent.svg');
+  background-size: cover;
+}
 
-  a {
-    margin-bottom: 32px;
-  }
+#atcl {
+  font-family: 'Mulish';
+  font-size: 32px;
 }
 
-@media screen and (max-width: 600px) {
```

Comment:
i removed this conditional styling since the styles that are changed are no longer used